src/frontend/src/pages/Calendar/SummaryModal/DesignReviewSummaryModal.tsx

```diff
@@ -0,0 +1,92 @@
+import { DesignReview } from 'shared';
+import NERModal from '../../../components/NERModal';
+import { Box, Grid, IconButton } from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import SaveIcon from '@mui/icons-material/Save';
+import { useState } from 'react';
+import { DesignReviewDelayModal } from './SummaryComponents/DesignReviewDelayModal';
+import StageGateWorkPackageModalContainer from '../../WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModalContainer';
+import DesignReviewSummaryModalDetails from './SummaryComponents/DesignReviewSummaryModalDetails';
+import DesignReviewSummaryModalCheckBox from './SummaryComponents/DesignReviewSummaryCheckbox';
+import DesignReviewSummaryModalButtons from './SummaryComponents/DesignReviewSummaryModalButtons';
+import DesignReviewSummaryModalAttendees from './SummaryComponents/DesignReviewSummaryModalAttendees';
+import { getTeamTypeIcon } from '../../CalendarPage/CalendarComponents/CalendarDayCard';
+import DesignReviewSummaryModalDetailsButton from './SummaryComponents/DesignReviewDelayModalDetailsButton';
+
+interface DRCSummaryModalProps {
```

Comment:
this stuff should go under the CalendarPage folder and the Calendar folder should be deleted